src/components/oppgaver/OppgaveView.tsx

```diff
@@ -472,7 +476,15 @@ const OppgaveView: React.FC<Props> = ({oppgave, oppgaverErFraInnsyn, oppgaveInde
             return;
         }
 
-        let formData = opprettFormDataMedVedleggFraOppgaver(oppgave);
+        try {
+            var formData = opprettFormDataMedVedleggFraOppgaver(oppgave);
+        } catch (e) {
+            dispatch(setOppgaveOpplastingFeilet(oppgave.oppgaveId, true));
+            logInfoMessage("Validering vedlegg feilet: " + e.message);
+            event.preventDefault();
+            console.log("DEBUG pcn: opprettFormDataMedVedleggFraOppgaver threw exception: " + e.message);
```

Comment:
DEBUG-logg

Reply:
Requester changes bare pga denne. Ellers ser det bra ut synes jeg 👍🏻

Reply:
Wops!

Reply:
Jeg er jaffal tydelig på at det er debug logg om ikke annet :-P

Reply:
Ja, lett å vite hvem som står bak også :D 
Litt vanskeligere for brukerne våre å forstå hvem de skal kontakte for å bli kvitt den i konsollen sin bare 😂

Reply:
Sant sant. Tenkte mest på at det er lettere å fange den opp i PRen. Og nå er den jo borte igjen.

Reply:
Ja, det var veldig lurt! 💯